lua/rest-nvim/curl/init.lua

```diff
@@ -114,6 +147,19 @@ local function create_callback(method, url)
 
     -- Send cursor in response buffer to start
     utils.move_cursor(res_bufnr, 1)
+
+    -- add syntax highlights for response
+    local syntax_file = vim.fn.expand(string.format("$VIMRUNTIME/syntax/%s.vim", content_type))
+
+    if vim.fn.filereadable(syntax_file) == 1 then
+      vim.cmd(string.gsub([[
+        unlet b:current_syntax
```

Comment:
this line triggers an error "unknown b:current_syntax" if I run a request twice

Reply:
I did push a small fix for this that checks if `b:current_syntax` exists a few seconds ago, please let me know if the issue persists.

Reply:
seems like that fixed it thx